docs/REST/throttling.md

```diff
@@ -5,9 +5,13 @@ Configuration stores have limits on the requests that they may serve. Any reques
 Throttling is divided into different quota policies:
 
 **TotalRequests** - total amount of requests/sec
-**TotalWrites** - total amount of write requests/second
+		
+**WriteRequests** - total amount of write and delete requests/second
```

Comment:
We can replace `write` by `update` to avoid confusion since a delete request is also a form of write request.

**WriteRequests** - total amount of update and delete requests/second

Reply:
Removed `and delete`